formats/financial_release/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "campaign_page"
+  ],
+  "properties": {
+    "campaign_page": {
```

Comment:
@jamiecobbett the campaign page is not a parent of financial release page. We are not actually using the link, but hardcoding it into the page body; we thought to put it in a links hash for documentation. 
Do you suggest that we remove it if it is only needed in the body content?